db/db_impl.cc

```diff
@@ -97,6 +97,7 @@
 
 namespace rocksdb {
 const std::string kDefaultColumnFamilyName("default");
+const std::string kPersistentStatsColumnFamilyName("_persistent_stats");
```

Comment:
@yiwu-arbug is correct. This is a case we should absolutely handle. It can be as simple as skipping this feature if we have a conflict name.

Also, "_persistent_stats" is too easy to conflict. We need to add more special characters to make it harder to conflict. "Persistent" doesn't need to appear in the CF name. Whatever written to the DB is persistent.